doc/utils/sphinx-config/conf.py.in

```diff
@@ -76,7 +76,7 @@ master_doc = 'Manual'
 
 # General information about the project.
 project = 'LAMMPS'
-copyright = '2003-2020 Sandia Corporation'
+copyright = '2003-2021 Sandia Corporation'
```

Comment:
Since conf.py is python code, you could just grab the current year and insert it automatically.